documentation-site/examples/modal/basic.js

```diff
@@ -23,8 +23,65 @@ export default () => {
         <ModalHeader>Hello world</ModalHeader>
         <ModalBody>
           Proin ut dui sed metus pharetra hend rerit vel non mi.
+          <br />
```

Comment:
do we need these additions? I think we already have a modal example with lots of text, called `Dealing with a buried interactive element `